lib/storage_keys.ts

```diff
@@ -25,6 +25,7 @@ export enum StorageKey {
   BiometricsState = 'biometrics_state',
   MobilePasscodeTiming = 'passcode_timing',
   MobileBiometricsTiming = 'biometrics_timing',
+  MobilePreferences = 'preferencesKey',
```

Comment:
Unless you need it this way for a reason, the key value should just be `preferences` instead of `preferencesKey`